02_bubble/sahidakhtar1/main_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMain(t *testing.T) {
+	//Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := "After sorting =  [1 2 3 5]\n"
+	actual := buf.String()
+	r.Equalf(expected, actual, "Unexpected output in main()")
+}
+
+var testData = []struct {
```

Comment:
You should probably define this data as local variable to `TestBubbleSort`

Reply:
Added `TestBubbleSortInput` which uses the same inout so not making any changes to it.